README.md

```diff
@@ -37,7 +37,18 @@ julia> with_logger(FormatLogger(LoggingFormats.JSON(; recursive=true), stderr))
 ```
 
 If it encounters something which does not have a defined `StructTypes.StructType` to use
-for serializing to JSON, it will fallback to converting the objects to strings, like the default `recursive=false` option does.
+for serializing to JSON, it will fallback to converting the objects to strings, like the default `recursive=false` option does. Handles the key `exception` specially, by printing errors and stacktraces using `Base.showerror`.
+
+```julia
+julia> f() = try
+                throw(ArgumentError("Bad input"))
+            catch e
+                @error "Input error" exception=(e, catch_backtrace())
+            end
+
+julia> with_logger(f, FormatLogger(LoggingFormats.JSON(; recursive=true), stderr))
+{"level":"error","msg":"Input error","module":"Main","file":"REPL[1]","line":4,"group":"REPL[1]","id":"Main_a226875f","kwargs":{"exception":["ArgumentError(\"Bad input\")","ArgumentError: Bad input\nStacktrace:\n  [1] f()\n    @ Main ./REPL[1]:2\n  [2] with_logstate(f::Function, logstate::Any)\n    @ Base.CoreLogging ./logging.jl:511\n  [3] with_logger(f::Function, logger::FormatLogger)\n    @ Base.CoreLogging ./logging.jl:623\n  [4] top-level scope\n    @ REPL[2]:1\n  [5] eval\n    @ ./boot.jl:373 [inlined]\n  [6] eval_user_input(ast::Any, backend::REPL.REPLBackend)\n    @ REPL ~/.asdf/installs/julia/1.7.0/share/julia/stdlib/v1.7/REPL/src/REPL.jl:150\n  [7] repl_backend_loop(backend::REPL.REPLBackend)\n    @ REPL ~/.asdf/installs/julia/1.7.0/share/julia/stdlib/v1.7/REPL/src/REPL.jl:244\n  [8] start_repl_backend(backend::REPL.REPLBackend, consumer::Any)\n    @ REPL ~/.asdf/installs/julia/1.7.0/share/julia/stdlib/v1.7/REPL/src/REPL.jl:229\n  [9] run_repl(repl::REPL.AbstractREPL, consumer::Any; backend_on_current_task::Bool)\n    @ REPL ~/.asdf/installs/julia/1.7.0/share/julia/stdlib/v1.7/REPL/src/REPL.jl:362\n [10] run_repl(repl::REPL.AbstractREPL, consumer::Any)\n    @ REPL ~/.asdf/installs/julia/1.7.0/share/julia/stdlib/v1.7/REPL/src/REPL.jl:349\n [11] (::Base.var\"#930#932\"{Bool, Bool, Bool})(REPL::Module)\n    @ Base ./client.jl:394\n [12] #invokelatest#2\n    @ ./essentials.jl:716 [inlined]\n [13] invokelatest\n    @ ./essentials.jl:714 [inlined]\n [14] run_main_repl(interactive::Bool, quiet::Bool, banner::Bool, history_file::Bool, color_set::Bool)\n    @ Base ./client.jl:379\n [15] exec_options(opts::Base.JLOptions)\n    @ Base ./client.jl:309\n [16] _start()\n    @ Base ./client.jl:495"]}}
```

Comment:
What do you think about scrubbing the backtrace? https://julialang.zulipchat.com/#narrow/stream/274208-helpdesk-.28published.29/topic/filtering.20backtraces

Reply:
I tried `Base.display_error` that does the scrubbing for us, what do you think?